tests/test_replykeyboardmarkup.py

```diff
@@ -96,7 +96,9 @@ def test_from_column(self):
         assert len(reply_keyboard_markup[1]) == 1
 
     def test_expected_values(self, reply_keyboard_markup):
-        assert isinstance(reply_keyboard_markup.keyboard, list)
+        assert isinstance(reply_keyboard_markup.keyboard, tuple)
+        for row in reply_keyboard_markup.keyboard:
+            assert isinstance(row, tuple)
```

Comment:
```suggestion
        assert all(isinstance(row, tuple) for row in reply_keyboard_markup.keyboard)
```
slightly more pythonic, unless this exceeds the line length, in which case leave it lol